instance/tests/test_newrelic.py

```diff
@@ -42,47 +42,28 @@ class NewRelicTestCase(TestCase):
     @responses.activate
     def test_get_synthetics_monitors(self):
```

Comment:
Since you renamed the function, it makes sense to rename this test function as well (i.e. remove the trailing "s").